src/credentials/CHIPCert.h

```diff
@@ -56,6 +56,8 @@ static constexpr uint32_t kMaxDERCertLength  = 600;
 // The decode buffer is used to reconstruct TBS section of X.509 certificate, which doesn't include signature.
 static constexpr uint32_t kMaxCHIPCertDecodeBufLength = kMaxDERCertLength - Crypto::kMax_ECDSA_Signature_Length_Der;
 
+CHIP_ERROR ASN1ToChipEpochTime(const chip::ASN1::ASN1UniversalTime & asn1Time, uint32_t & epochTime);
```

Comment:
Why is this being added?  It's also declared below, and not used before the declaration there, right?